vllm/worker/habana_model_runner.py

```diff
@@ -1467,6 +1467,11 @@ def warmup_model(self, kv_caches: List[torch.Tensor]) -> None:
                     len(self.decode_buckets),
                     list(sorted(self.decode_buckets)))
 
+        if not htorch.utils.internal.is_lazy() and not self.enforce_eager:
+            cache_size_limit = len(self.prompt_buckets) + len(self.decode_buckets)
+            torch._dynamo.config.cache_size_limit = cache_size_limit
```

Comment:
maybe we have to check if user already do not setup greater values then we are setting. Some think like  limit = max (limit, new_value)

Reply:
Thanks. I think this should be more reasonable. I will modify the code.